src/webview-scripts/CallGraphRenderer.ts

```diff
@@ -137,7 +188,49 @@ export class CallGraphRenderer {
         this.render();
     }
 
+    timer: NodeJS.Timeout
+    visited: CallGraphLayoutNode[] = []
+
+    linkTarget = (hop: number, nodes: CallGraphLayoutNode[]) => {
+        this.visited.push(...nodes)
+        const kids: CallGraphLayoutNode[] = [];
+        this.linkSelection
+            .classed("link-source", (link) => {
+                const n = this.visited.find(n => n === link.source)
```

Comment:
Formatting: trailing semicolons missing in this part of the code. VS Code can add them automatically, if enabled...